### ISSUE TITLE
Request - remove part attribute container volume

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8896]
Help us understand your request (check below):

- [x] other

**Describe what you're trying to do** 

https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_attribute_type#container_volume doesn't make sense to me - looks like lost container data - doesn't have a functional definition, is inspiring requests for more funky data, and wasn't added under normal/modern procedures. I propose we 

1. Remove it, or
2. Document it in some way that involves a functional need
 
Term: https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_attribute_type#container_volume

Probable source: https://github.com/ArctosDB/arctos/issues/3841

Inspired funkiness: https://github.com/ArctosDB/arctos/issues/8895

Usage:

```
 guid_prefix |  c   
-------------+------
 ALMNH:Inv   | 3184
 MSB:DGR     | 2678
 MSB:Herp    |    1
 MSB:Mamm    |   14
 MSB:Para    | 6910
 UMNH:Mamm   |   16
 UMNH:Para   | 1042
```

**Comment by Jegelewicz** [comment]
I don't think this is a working group discussion. There are a limited number of users and they should weigh in before we present this to the community.

@dustymc can you add the potentially affected users here?
